README.md

```diff
@@ -1,11 +1,11 @@
 We're hiring!
 --------
-Ever thought about joining us?<br/>
```

Comment:
Are you sure? The preview still shows it as being broken line by line and I've never come across .md *needing* the explicit tag, if it's being used somewhere else and that's why it needs it, let me know and I'll add it in :+1: